presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -56,6 +56,8 @@ public void testGrantRevoke()
         queryExecutorForAlice.executeQuery(format("GRANT INSERT, SELECT ON %s to bob", tableName));
         assertThat(queryExecutorForBob.executeQuery(format("INSERT INTO %s values ('t', 3)", tableName))).hasRowsCount(1);
         assertThat(queryExecutorForBob.executeQuery(format("SELECT * from %s", tableName))).hasRowsCount(1);
+        assertThat(() -> queryExecutorForBob.executeQuery(format("DROP TABLE IF EXISTS %s", tableName))).
+                failsWithMessage(format("Access Denied: Cannot drop table default.%s", tableName));
```

Comment:
Can you do the same for REVOKE? ie: after revoking INSERT, the user is still able to SELECT from the table